tsconfig.fetch.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+      "moduleResolution": "node",
+      "module": "esnext",
```

Comment:
are there any compatibility issues with using a different module system we use on the consuming packages (e.g. marketing site)? Marketing site is still using commonJS and now we're using node for this typescript client package

Reply:
The marketing site is actually using the same [module type](https://github.com/OffTheRecordApp/otr_marketing_site/blob/42d80fb5dae3627f31bd119831f1e070191ef889/tsconfig.json#L4). [Webpack will automatically make modules work in older browsers](https://webpack.js.org/guides/getting-started/#modules). The webpack documentation warns against using `commonjs` and requires the `ES2015` `import / export` syntax to enable tree shaking. 

`esnext` would probably be fine.. but for peace of mind I updated the module type to the lowest one that supports `dynamic imports` on the marketing site: https://github.com/OffTheRecordApp/otr_marketing_site/pull/399